maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -281,10 +281,11 @@
 
     /**
      * The test source directory containing test class sources.
+     * Important for TestNG HTML reports.
```

Comment:
please add that `only` for ... and maybe bold/italic such sentence

Reply:
I wanted to initially do write so :-)

Reply:
@slawekjaranowski 
done, pls check it again